angr/engines/light/data.py

```diff
@@ -63,6 +63,18 @@ def try_unpack_const(expr):
 
 
 class RegisterOffset:
+    """
+    Represents a symbolic integer that equals to the given offset added the symbolic register's value.
+
+    For example, `RegisterOffset(4, eax, 8)` represents a (symbolic) integer between 0 and 15, that equals to `eax + 8`
+    regardless of what value `eax` holds.
+    Note that it does not matter whether `eax + 8` points to a valid memory location or not.
+
+    :param bits:                     The size of the resulting value, in number of bits.
+    :param string reg:               The register's name from which the value will be used to compute the offset from.
+    :param int|claripy.BVS offset:   The value of the offset. A concrete offset is modeled as an <int>, while a
```

Comment:
1) the syntax is `a or b`
2) claripy.BVS isn't a type. you want claripy.ast.BV

Reply:
1. Not sure where you have seen this syntax documented, but it gives me odd results.

The doc parser seems to rely on spaces to identify tokens...

Used [`Union[int,claripy.BVS.offset]`](https://docs.python.org/3.7/library/typing.html#typing.Union).